modules/ROOT/pages/provisioning-openstack.adoc

```diff
@@ -0,0 +1,97 @@
+= Provisioning Fedora CoreOS on OpenStack
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on an
+OpenStack cloud environment, either private, or public (like https://vexxhost.com/[VEXXHOST]).
+
+== Prerequisites
+
+Before provisioning a FCOS machine, you must have an Ignition configuration file
```

Comment:
Aside: looks like we inconsistently use "a FCOS" and "an FCOS" throughout the docs. We should decide on one and clean up the docs in a follow-up.

Reply:
I'm cool with that. Want to open an issue?

Reply:
changed to `an FCOS`